scripts/errcheck.sh

```diff
@@ -0,0 +1,28 @@
+# Check gofmt
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+echo "==> Checking for unchecked errors..."
+
+if ! which errcheck > /dev/null; then
+    echo "==> Installing errcheck..."
+    go get -u github.com/kisielk/errcheck
+fi
+
+err_files=$(errcheck -exclude scripts/errcheck_excludes.txt \
```

Comment:
The bin location might not always be in the PATH, but it's easy to reference the binary directly like this:
```suggestion
err_files=$($(go env GOPATH)/bin/errcheck -exclude scripts/errcheck_excludes.txt \
```